### ISSUE TITLE
Rain on visor effect isn't lined up for each eye

### ISSUE THREAD

**Issue by SpikeEscape** [Raicuparta/NomaiVR#63]
on giants deep for example, whenever there are water droplets on the players helmet they are displayed at exactly the same screen positions on each eye, causing you to see double in VR.

I'm not sure if it's possible to edit the positions of that effect for each eye individually in order to apply proper depth, but it's worth looking into

**Comment by Raicuparta** [comment]
Interesting, never paid attention to it. Maybe I unknowingly disabled the effect by lowering the graphics settings.

The problem with "seeing double" is the same as some UI elements I've fixed: they are just too close to your eyes (some times just a millimeter or two between the effect and the camera). For UI elements the solution is to just move these elements far from the camera, but for this effect it might not really make sense. I will play with some distances and see what feels best. As a last resort, will just have to disable it.

**Comment by Raicuparta** [comment]
@SpikeEscape what version did you test? Was it 0.6 or a later one? In 0.7 I increased the gap between the camera and the helmet, so this would make things slightly better.

I just tested it myself and it seems fine? The rain effect is exactly where the helmet visor is, which makes sense. Of course you'll see double because you're looking through the visor to focus on something far away. But that's what would happen in real life too.

If you have seen the effect in 0.7+, let me know if you think this is something that bothers you. I can make a note to make the effect toggleable.
